tests/e2e/features/login.feature

```diff
@@ -10,7 +10,7 @@ Feature: login feature
 
 
     Scenario: user logs in with valid credentials
-        When the user logs in with valid username "user" and valid password "aaaa1234"
+        When the user logs in with username "user" and password "aaaa1234"
         Then the user should be navigated to the search page
```

Comment:
```suggestion
        Then the user should be navigated to the search page
```
The step seems weird, after login user redirects to the search page.

Reply:
you are right it should be homepage instead.